Make gis visualization work with huge tables

M393 opened the pull request:
The query was executed without a limit clause but only the meta data was read, so adding a 'limit 0' clause is a big improvement for big tables.

Without this change I could not open the visualization for a big table.

M393 committed:
Make gis visualization work with huge tables

The query was executed without a limit clause but only the meta data
was read, so adding a 'limit 0' clause is a big improvement for big tables.

Signed-off-by: Maximilian Krög <maximilian.kroeg@geocept.com>

M393 commented on libraries/classes/Controllers/Table/GisVisualizationController.php:
```diff
@@ -200,4 +186,37 @@ private function saveToFile(string $filename, string $format): void
     }
+
+    /**
+     * @return list{list<string>,list<string>}
```
Thanks!

kamil-tekiela reviewed (changes requested):
Code comments should ideally describe why something is being done and not what is being done.

M393 commented:
These comments were not from me, I just moved the code to the new function and added this part:
```php
        $parser = new Parser($sqlQuery);
        /** @var SelectStatement $statement */
        $statement = $parser->statements[0];
        $statement->limit = new Limit(0, 0);
        $limitedSqlQuery = $statement->build();
```
That being said, I do agree, that the comments were not very helpful, so I removed them.
